Gruntfile.js

```diff
@@ -186,8 +201,8 @@ module.exports = function (grunt) {
     filerev: {
       dist: {
         src: [
-          '<%= yeoman.dist %>/scripts/{,*/}*.js',
-          '<%= yeoman.dist %>/styles/{,*/}*.css',
+          // '<%= yeoman.dist %>/scripts/{,*/}*.js',
+          // '<%= yeoman.dist %>/styles/{,*/}*.css',
```

Comment:
The code previously commented out was causing problems, but in the follow-up changes just submitted, I've applied the same workaround for the corresponding issue in generator-angular doumented here: https://github.com/yeoman/generator-angular/issues/684

Since the filever plugin is now communicating properly with usemin, I was able to put those lines of code back.